doc/decisions/decision_process.md

```diff
@@ -24,24 +26,42 @@ Substantial decisions must be made in a transparent and participative way.
   - Rebase only if the decision was already accepted and has a merge conflict.
 - For reviewers:
   - Prefer to directly give suggestions how to change sentences.
-  - General questions should be asked in the root of "Conversation" and not at random sentences in the review.
+  - General questions should be asked in the root of "Conversation" and not at vaguely related sentences in the review.
+- Decision PRs do not significantly change anything but one decision.
+- Changes not changing the decision step or the direction of the decision are not decision PRs.
+- The person merging the decision PR must be someone else as the person that created the decision.
+- The purpose of decisions is to have clear descriptions of technical problems and solutions.
+  There is no claim that decisions contain everything that was said.
+  In particular corrections of wrong decision text is, if at all, only visible via git history.
+  Rather it is important that decisions:
+  - contain everything relevant, and
+  - what is written is technically correct.
+- Participants are explicitly allowed to skip discussions and only read the decisions text.
+  If the problem is not yet clear, only partial reviews are encouraged.
+  @markus2330 will do so of time budget reasons and to make sure the decisions stay understandable.
 
 ## Assumptions
 
 - People want to be informed about or even participate in what Elektra looks like in the future.
 - People writing or reviewing decisions want Elektra to improve, so they also want to accept (acceptable) decisions.
-  In general they also want change if it brings Elektra towards its [goals](/doc/GOALS.md) (but doesn't violate Elektra's stability guarantees).
-- We will always be able to reach an consensus.
+  In general people want change if it brings Elektra towards its [goals](/doc/GOALS.md).
+- We will always be able to reach an consensus even if it requires that the core or plugins get multiple implementations.
   We don't need a vote (besides the approved review) or a benevolent dictatorship.
 - Unlike the Rust Decision process, decisions in Elektra do not have a disadvantage if they were flawed in early stages.
   Only the end results counts.
 - Different to initiatives like Rust, most contributors in Elektra are not experts in configuration management or programming languages.
   So we do not expect that a clear problem or solution is in the decision writer's mind beforehand.
   Instead the decision process is a supported learning process.
+- People focus on getting the best solutions and not to wish for the impossible.
+- After 1.0, due to stability guarantees, not so many decisions are required anymore.
```

Comment:
I'm not so sure about that. Things could be added as entirely new API without touching the existing parts. Why would that require less of a decision process? Also I think the stability guarantees would just _increase_ the need for a full decision process, if we want to change something. It is much harder to find a correct solution, if we have to think about stability as well.

Reply:
I agree, I completely changed that: c1a3d5c9341d24b6391c49cf098a237485ede18c